### PR TITLE
Tweak install.rdf values and icon

### PR DESCRIPTION
**Pull request by pdehaan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,13 +62,13 @@
   "scripts": {
     "build": "bash ./bin/xpi.sh",
     "eslint": "eslint addon test --ext jsm --ext js --ext json",
+    "firefox": "export XPI=dist/linked-addon.xpi && npm run build && node run-firefox.js",
+    "harness_test": "export XPI=dist/linked-addon.xpi && mocha test/functional_tests.js --retry 2 --reporter json",
```
Comment (pdehaan) [inline package.json]: These were just an A-Z shift when it ran **fixpack**.

File: package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
   "addon": {
     "$ABOUT": "use these variables fill the moustache templates",
     "id": "tracking-protection-messaging@shield-study.mozilla.com",
-    "name": "Button Icon Preference Study (Shield Study Example)",
+    "name": "Tracking Protection Messaging Shield Study",
```
Comment (pdehaan) [inline package.json]: Now that this has a more accurate name, the `description` field below (L16) feels redundant and less than informative (it's now the same value as `name`).

Comment (biancadanforth) [reply package.json]: My [earlier PR](https://github.com/biancadanforth/tracking-protection-shield-study/pull/53) updates the description field to say: "The purpose of this study is to test various types of messaging around Tracking Protection in Firefox."

**Comment by pdehaan** [comment]
Not sure anything else is worth rebasing. I think the only changes were adding .DS_Store to .gitignore and a minor TODO tweak in ESLint config.

**Comment by biancadanforth** [comment]
I'll roll the diff into my next patch.

**Pull request closed without merging**